src/index.js

```diff
@@ -632,6 +691,11 @@ const handler = (event, context) => {
   if (event.Records) {
     handleRecords(event, context, lintEnv);
   }
+
+  if (process.env.EventBridge_Logs === 'true') {
```

Comment:
Iam just thinking is this really required?
cant we just do the below,.,,
if (event.source == "aws.athena") {
}